pkg/netpol/eval/internal/k8s/netpol.go

```diff
@@ -141,8 +143,8 @@ func (np *NetworkPolicy) ruleConnections(rulePorts []netv1.NetworkPolicyPort, ds
 	return res, nil
 }
 
-// isRepresentativePod determines if the peer's source is representativePeer; i.e. its pod fake and has RepresentativePodName
-func isRepresentativePod(peer Peer) bool {
+// hasRepresentativePod determines if the peer's source is representativePeer; i.e. its pod fake and has RepresentativePodName
+func hasRepresentativePod(peer Peer) bool {
```

Comment:
rename `hasRepresentativePod` to `isPeerRepresentative` ?

Reply:
done